docs/guides/key-manager/execute-relay-call.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_label: 'Execute Relay Call'
+sidebar_position: 2
+---
+
+# Execute Relay Call
+
+The [LSP6 KeyManager standard](../../standards/universal-profile/lsp6-key-manager.md) enables anybody to execute a transation on behalf of a Universal Profile, given they have a valid transaction which has been signed by a key which controls the Universal Profile.
```

Comment:
_Suggested changes._

I think also maybe would be good to link to the **permissions** section on the LSP6 Standard page for the word "permissions" at the end (I cannot find the correct link).


```suggestion
The [LSP6 KeyManager standard](../../standards/universal-profile/lsp6-key-manager.md) enables any `address` (EOA or contract) to execute a transaction on behalf of a Universal Profile, given they have a valid signature for a transaction and have the right permissions on the Universal Profile.
```